rs/boundary_node/ic_boundary/src/snapshot.rs

```diff
@@ -243,6 +295,11 @@ impl Snapshotter {
             .context("failed to get subnet ids")? // Result
             .context("subnet ids not available")?; // Option
 
+        // Fetch a list of API BNs
+        let api_bns = self
+            .get_api_boundary_nodes(version)
```

Comment:
We need just the count of API BNs right? Then, I would simply reuse existing code and avoid creating a new function:
```
        let version = self.registry_client.get_latest_version();
        let api_bns = self
            .registry_client
            .get_api_boundary_node_ids(version)
            .context("unable to get API BNs")?;
```
and just store `pub api_bns: Vec<NodeId>` in the `RegistrySnapshot`

Reply:
Yes, for now we just count them, but for the future sake I've created an `ApiBoundaryNode` struct with some subset of fields.

Yes, thanks, I'll use the `.get_api_boundary_node...()` functions, probably I've missed them and reimplemented redundantly.